CMakeLists.txt

```diff
@@ -72,6 +72,9 @@ set(GENERATE_PROTOBUF_FILES_IN_BUILD_DIR True CACHE BOOL "Generate all protobuf
 set(USE_LIBCSV False CACHE BOOL "Disable library components that import data from csv files")
 set(LIBCSV_DIR "" CACHE PATH "Path to libcsv header and library")
 
+#Testing
+option(DISABLE_TESTING "Disable testing of builds") 
```

Comment:
Should this default to `ON`? Or do we intend for people to opt-out of testing rather than opt-in?

Reply:
I think we should allow folks to opt-out of testing, default should allow for testing as GenomicsDB is meant to be built by developers. But I like the way you handled it in OmicsDS where if Catch2 is not installed, it gets disabled by default. Catch2 is still a submodule in GenomicsDB btw, we need to start using installed Catch2 instead.